src/Codeception/Module/AMQP.php

```diff
@@ -325,9 +336,9 @@ public function seeMessageInQueueContainsText($queue, $text)
      *
      * @return int
      */
-    public function _countMessage($queue)
+    public function countMessage(string $queue): int
```

Comment:
`_countMessage` method couldn't be called from test code, but it could be called by helper code (This is Codeception convention).
Renaming would make it equal to other methods.